### DISCUSSION TITLE
How to use Prisma's CRUD in Nuxt/Vue

### DISCUSSION THREAD

**Discussion by bastianhilton** [prisma/prisma#16315] [Q&A]
Hello i currently am using Prisma within Nuxt, so far it works as expected until I want to perform crud tasks. The documentation on the website here: [https://www.prisma.io/docs/concepts/components/prisma-client/crud#update](url) shows snippets but not really how to use them within a page.

This is my code:

```ts
import prisma, {update} from '../../../../plugins/extensions/prisma.client'

async function agreementUpdate(){
  const updateAgreement = await prisma.agreements.update({
  where: {
    id: 6,
  },
  data: {
    name: "",
    excerpt: "",
    type: "",
    content: "",
    image: "",
    reference_id: "",
    user_id: "",
    shop_id: ""
  },
})
}

agreementUpdate();
```

It is giving me an error saying Prisma/client cannot be used within the browser. Any ideas on how to perform dynamic crud operations within Nuxtjs.

**Comment by nurul3101** [comment]
Hey @bastianhilton 👋 

Can you have a look at this official [Prisma Example](https://github.com/prisma/prisma-examples/tree/latest/javascript/rest-nuxtjs) of using Prima with Nuxt and Vue?
This should be helpful in getting this working.
Let me know if you run into any issues.

**Reply by pantharshit00** [reply]
👋 Hello @bastianhilton,  did you perhaps have a chance to check @nurul3101's recommendation? Let us know if you still have any questions!
 
 Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
